ui/src/app/views/settings/component.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import { mount } from 'enzyme';
+import * as React from 'react';
+
+import {SettingsReact, SettingsState} from './component';
+
+import {Cluster, ClusterStatus} from 'generated/fetch/api';
+
+describe('SettingsComponent', () => {
+  let props: {};
+
+  const component = () => {
+    return mount<SettingsReact, {}, SettingsState>
+    (<SettingsReact {...props}/>);
+  };
+
+  beforeAll(() => {
+    const popupRoot = document.createElement('div');
+    popupRoot.setAttribute('id', 'popup-root');
+    document.body.appendChild(popupRoot);
+  });
+
+  afterAll(() => {
+    document.removeChild(document.getElementById('popup-root'));
+  });
+
+  beforeEach(() => {
+    props = {};
```

Comment:
Doesn't seem like there's much purpose to this - or are you setting up the convention / boilerplate for future tests?

Reply:
Setting up the boilerplate, but I don't think its actually important to build that out. Will remove.